fard_resources/assets/fard/lang/de_de.json

```diff
@@ -0,0 +1,16 @@
+{
+    "%1$s": "%2$s",
+
+    "pack.fard.name": "FARD",
+    "pack.fard.prefix": "[FARD] ",
```

Comment:
Can be removed in all but `en_us.json` as they are the same for all languages

Reply:
Done.